config.js

```diff
@@ -144,8 +105,30 @@ exports.updateFromArgv = function(argv) {
     // console messages
     // C - test exports warnings
 
-    testExportsWarnings();
+    warnIfNopauseIsMissing("ranked");
+    warnIfNopauseIsMissing("unranked");
+    console.log("\n");
+
+}
+
+function exportInitialConfig() {
+    exportInitialConfigRankedUnranked("");
+    exportInitialConfigRankedUnranked("_ranked");
+    exportInitialConfigRankedUnranked("_unranked");
 
+    exports.start_date = new Date();
+}
+
+function exportInitialConfigRankedUnranked(rankedUnrankedUnderscored) {
```

Comment:
I think the name `exportInitialConfig` would be good enoug.

Reply:
but we already use this name at line 114 for the wrapper function, also exportInitialConfig of line 114 does other things than just the _ranked exports, it also exports start date for example, and there we could add other initial config exports in the future

so maybe exportInitialConfigUnderscored for the sub functions?

Reply:
fixed, it is indeed clearer ranked unranked could be any option, but this function is only for the underscored (`_ranked`, `_unranked`, ``) initial exports

Reply:
Ah yes i missed the other function. But i like the new name, nicely found :D